.github/CODEOWNERS

```diff
@@ -1374,6 +1376,7 @@ x-pack/test/api_integration/deployment_agnostic/services/ @elastic/appex-qa
 x-pack/test/**/deployment_agnostic/ @elastic/appex-qa #temporarily to monitor tests migration
 
 # Core
+/x-pack/test/api_integration/apis/logstash @elastic/kibana-core
```

Comment:
It looks like the APIs tested here are owned by `@elastic/logstash` as per https://github.com/elastic/kibana/blob/b67ab78a651cc7b490b35cc74e3ee2d14c9be0ae/x-pack/plugins/logstash/kibana.jsonc#L4

Reply:
What is the impact of this change on the Logstash team? - we have little capacity or ability to do work in Kibana at the moment

Reply:
> What is the impact of this change on the Logstash team? - we have little capacity or ability to do work in Kibana at the moment

@afharo this is one reason why I elected to assign this to the core team

Reply:
IMO, the plugin is already owned by the logstash team, so... unless we expect big breaking changes in the FTR suite, I wouldn't expect them to require any changes if the plugin doesn't change.

@wayneseymour, am I missing any other potential effort in maintaining these tests?

Reply:
Back to the logstash team: https://github.com/elastic/kibana/pull/196919/commits/55ced3dc6845822c4702fafdcb82dccabb40dfbb